documentation/mkdocs.yml

```diff
@@ -29,6 +29,15 @@ nav:
         - 'SAP Cloud SDK pipeline':
             - 'Introduction': pipelines/cloud-sdk/introduction.md
             - 'Configuration': pipelines/cloud-sdk/configuration.md
+            - 'Stages':
+                - 'Build and Test': stages/build.md
+                - 'Frontend Unit Tests': stages/additionalunittests.md
+                - 'Backend and Frontend Integration Tests': stages/integration.md
+                - 'End to End Tests': stages/acceptance.md
+                - 'Security Stage': stages/security.md
+                - 'Performance Stage': stages/performance.md
+                - 'Artifact Deployment': stages/promote.md
```

Comment:
For Artifact Deployment, we use piperPipelineStageArtifactDeployment. I think that one doesn't have an .md file yet, as I was not aware of those at the time.

Reply:
thanks, that was a bit unclear and I made the wrong assumption.